app/main/views/send.py

```diff
@@ -262,6 +264,23 @@ def set_sender(service_id, template_id):
     )
 
 
+def remove_notify_from_sender_options(sender_details):
+    # Remove US Notify/Notify.gov from users list of sender
+    # options during message send flow
+    sender_details = [
+        sender for sender in sender_details if verify_sender_options(sender)
+    ]
+
+    return sender_details
+
+
+def verify_sender_options(sender):
+    if sender.get("sms_sender") not in ["Notify.gov", "US Notify"]:
+        return True
+
+    return False
```

Comment:
Whoops, it looks like one of the checks went missing!

```suggestion
    if sender.get("sms_sender") in ["Notify.gov", "US Notify"] and sender["is_default"]:
        return True

    if sender.get("sms_sender") not in ["Notify.gov", "US Notify"]:
        return True

    return False
```